src/components/Home/ContainerTwo/styles.module.scss

```
@@ -0,0 +1,45 @@
+.cont2{
```

Ten en cuenta los espaciados.
```suggestion
.cont2 {
```